plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/AdvancedConnectionEditor.java

```diff
@@ -287,10 +289,14 @@ protected GridLayoutFactory adjustAdvancedCompositeLayout(GridLayoutFactory grid
 		return composite;
 	}
 
-	private void discoverRegistryPressed() {
+	private void discoverRegistryPressed(Shell shell) {
 		String ret = RegistryProviderModel.getDefault().getRegistryURL(pageModel.getSelectedConnection());
 		if( ret != null ) {
 			registryURLObservable.setValue(ret);
+		} else {
```

Comment:
Can we report this to the UI in a more UX way: I'm thinking about reporting an Info status to the databinding framework would be more user friendly